browser/ephemeral_storage/BUILD.gn

```diff
@@ -0,0 +1,17 @@
+source_set("ephemeral_storage") {
+  check_includes = false
```

Comment:
curious, does it fail `gn check`?

Reply:
`npm run gn_check` does not give any errors.

Reply:
yeah then why add `check_includes = false`?

Reply:
Good question. Looks like there is a circular dependency which is used to include `profile.h` in the tab helper.

Reply:
we have a master of Deps @Gyuyoung who could help :) 
in any case, we should only add `check_includes = false` together with a link to a github issue

Reply:
@iefremov this cannot be resolved until the `//chrome/browser` dep issue has been resolved and @Gyuyoung is still working on that

Reply:
yep, so I just want to have an issue number to be mentioned here

Reply:
Is this an example of @Gyuyoung fixing a similar issue? https://github.com/brave/brave-core/pull/6712/files If so, I can follow the same pattern here.

Reply:
The addition of `PerTLDEphemeralStorage` in the tab helper makes this dependency unnecessary now, so I should be able to remove the `check_includes = false` line.